vbaSrc/Employee

```diff
@@ -0,0 +1,54 @@
+VERSION 1.0 CLASS
+BEGIN
+  MultiUse = -1  'True
+END
+Attribute VB_Name = "Employee"
+Attribute VB_GlobalNameSpace = False
+Attribute VB_Creatable = False
+Attribute VB_PredeclaredId = False
+Attribute VB_Exposed = False
+Option Explicit
+
+Public name As String
+Public surname As String
+Public organisationalUnit As String
+Public supervisorName As String
+Public supervisorSurname As String
+Public contractStartDate As String
+
+Sub gatherInfo()
+On Error GoTo cancel
+    name = getInput("Podaj imi�")
+    surname = getInput("Podaj nazwisko")
+    organisationalUnit = getInput("Podaj dzia� (np. YIAID)")
```

Comment:
Can you try to fix encoding?

Reply:
The vbe expects this (Windows) encoding. All file related operations (input, output, exporting modules etc.) use it.

Reply:
In short - even if it is possible, it would be a massive hack.

Reply:
As we talked offline - switched to english to avoid issues with encoding alltogether.